metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/datahub_listener.py

```diff
@@ -5,23 +5,25 @@
 import threading
 from typing import TYPE_CHECKING, Callable, Dict, List, Optional, TypeVar, cast
 
-import airflow
```

Comment:
Have you run lint on this PR?

Reply:
yes.. I executed lint and as a part of that only this files were modified